### ISSUE TITLE
Windows 64 bit installer crashes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [maximemaher/pyodbc#151]
```
What steps will reproduce the problem?
1. Download pyodbc-2.1.8.win-amd64-py2.7.exe from this website.
2. Run under Windows 7 Ultimate 64 bit.

What is the expected output? What do you see instead?
Expected installation of pyodbc module in Python 2.7.1 (r271:86832, Nov 27 
2010, 17:19:03) [MSC v.1500 64 bit (AMD64)].
Instead, the installer crashes.


What version of the product are you using? On what operating system?
See above.

Please provide any additional information below.
Files that help describe the problem: 
d:\Users\Tobias\AppData\Local\Temp\WERCAA8.tmp.WERInternalMetadata.xml
d:\Users\Tobias\AppData\Local\Temp\WERF66.tmp.appcompat.txt
d:\Users\Tobias\AppData\Local\Temp\WERF76.tmp.hdmp
are attached.

Let me know if I can provide any additional info.

Best, Tobias


```

Original issue reported on code.google.com by `tbss...@googlemail.com` on 25 Jan 2011 at 10:37

Attachments:
- [WERCAA8.tmp.WERInternalMetadata.xml](https://storage.googleapis.com/google-code-attachments/pyodbc/issue-151/comment-0/WERCAA8.tmp.WERInternalMetadata.xml)
- [WERF66.tmp.appcompat.txt](https://storage.googleapis.com/google-code-attachments/pyodbc/issue-151/comment-0/WERF66.tmp.appcompat.txt)
- [WERF76.tmp.hdmp](https://storage.googleapis.com/google-code-attachments/pyodbc/issue-151/comment-0/WERF76.tmp.hdmp)

**Comment by GoogleCodeExporter** [comment]
```
I have not been able to reproduce this or to discover what could be causing it.

If you need a temporary work around, all the installer does is copy pyodbc.pyd 
into the Python lib\site-packages directory.  If you can successfully install 
it on any machine, you can then copy the file to the machine you need it on.

Now, the interesting thing is I don't actually write any code for the 
installer, so it should be using the same code as all other Python x64 
installers.  Have you installed any others that worked?
```

Original comment by `mkleehammer` on 6 Feb 2011 at 7:40
- Changed state: **Investigating**

**Comment by GoogleCodeExporter** [comment]
```
I'm also hitting this issue on Win7 Enterprise 64 bit.
```

Original comment by `Symmetri...@gmail.com` on 22 Apr 2011 at 7:37

**Comment by GoogleCodeExporter** [comment]
```
I've managed to resolve this issue by running the pyodbc installer as 
Administrator. 

The installer did not ask for admin rights when I first ran it; is this 
something that you have control over or is this an issue with all Python 
installers?
```

Original comment by `Symmetri...@gmail.com` on 25 Apr 2011 at 8:46

**Comment by GoogleCodeExporter** [comment]
```
I think this is fixed in 2.1.9-beta15.  Same as issue 185

```

Original comment by `mkleehammer` on 11 Jul 2011 at 12:31
- Changed state: **Complete**
